CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Both the `Histogram` and `HistogramDataPoint` are redefined with a generic argument of `[N int64 | float64]` in `go.opentelemetry.io/otel/sdk/metric/metricdata`. (#3849)
 - The metric `Export` interface from `go.opentelemetry.io/otel/sdk/metric` accepts a `*ResourceMetrics` instead of `ResourceMetrics`. (#3853)
 
+### Fixed
+
+- Panic in Prometheus exporter on concurrent `Collect`. (#3899)
```

Comment:
Maybe

```suggestion
- Fix a race condition in `go.opentelemetry.io/otel/exporters/metric/prometheus` that could cause a panic. (#3899)
```

1. The `func (c *collector) Collect` is not an publicly visible method 
2. Reference the package instead of name
3. The statement should say what has changed (and not describe the issue)

Reply:
Updated, thanks.